USERS.md

```diff
@@ -130,7 +130,8 @@ Currently, the following organizations are **officially** using Argo CD:
 1. [Whitehat Berlin](https://whitehat.berlin) by Guido Maria Serra +Fenaroli
 1. [Witick](https://witick.io/)
 1. [Yieldlab](https://www.yieldlab.de/)
-1. [Sap Labs](http://sap.com)
-1. [Smilee.io](https://smilee.io)
-1. [Metanet](http://www.metanet.co.kr/en/)
-1. [Unifonic Inc](https://www.unifonic.com/)
+2. [Zimpler](https://www.zimpler.com/)
```

Comment:
Thank you for adding @r0bobo ! Can you please change the numbering to `1.` ?

Reply:
Right, sorry about that. (Damn you autoformat!)

Reply:
thank you!